package.json

```diff
@@ -12,8 +12,8 @@
     "start:holofuel-live": "REACT_APP_HOLOFUEL_APP=true REACT_APP_RAW_HOLOCHAIN=true npm run start:live",
     "build": "node scripts/build.js",
     "build:holofuel": "REACT_APP_HOLOFUEL_APP=true REACT_APP_RAW_HOLOCHAIN=true node scripts/build.js",
-    "hc:start-manual-1": "holochain -c conductor-config.toml",
-    "hc:start-manual-2": "holochain -c conductor-config-extra.toml",
+    "hc:start-manual-1": "HC_IGNORE_SIM2H_URL_PROPERTY=true holochain -c conductor-config.toml",
+    "hc:start-manual-2": "HC_IGNORE_SIM2H_URL_PROPERTY=true holochain -c conductor-config-extra.toml",
```

Comment:
Nice call. :100: I kept debating whether this was appropriate to add, but after more reflection I do feel like this best supports the dev env.